tox.ini

```diff
@@ -47,7 +47,7 @@ commands = {[testenv:build]commands}
 # Envs that will only be executed on CI that does coverage reporting
 [testenv:coverage]
 commands =
-    coverage run test.py
+    coverage run test.py -f -p
```

Comment:
I suggest to use the long argument ``--form``  and ``--pynwb``

Reply:
@jcfr I changed it to use the long argument names. Is it good now?